README.md.tpl

```diff
@@ -468,7 +486,7 @@ Alias: `f-dir?`
 
 {{f-same?}}
 
-Alias: `f-equal?`
+Alias: `f-same-p f-equal? f-equal-p`
```

Comment:
You typed _"Aliases"_ above when there are multiple aliases, please change this as well.